perf/terraform/configs/local/terraform.tf

```diff
@@ -39,7 +39,7 @@ variable "ci_enabled" {
 variable "long_lived_enabled" {
   type        = bool
   description = "Whether or not to create long lived resources (in CI, used across runs; e.g. VPCs)"
-  default     = false
+  default     = true
```

Comment:
Let's revert this before merge.

Reply:
This change will make CI fail. It will try to spin up the long-lived resources which already exist in the libp2p AWS account. Mind reverting @maschad?